ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/Init.hs

```diff
@@ -37,12 +38,11 @@ openMempool ::
   => ResourceRegistry m
   -> LedgerInterface m blk
   -> LedgerConfig blk
-  -> MempoolCapacityBytesOverride
+  -> TxOverrides blk
```

Comment:
Currently, the haddocks of `TxOverrides` say that it is about overriding the size of forged blocks (only enabling to make blocks *smaller* than allowed by the ledger):

https://github.com/IntersectMBO/ouroboros-consensus/blob/f2f942b4279582a4a5b296c04698d0a2e9c12c82/ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/Capacity.hs#L58-L76

As mentioned by you already on Slack, this might not be expressive enough (you can't increase the capacity); [here](https://input-output-rnd.slack.com/archives/CDA6LUXAQ/p1719878861071389?thread_ts=1719870923.905459&cid=CDA6LUXAQ), you asked

> Do you think "how many block's worth of tx" would be equally useful as the current "how many bytes of tx"?

which was positively affirmed. So what about instead of using `TxOverrides` here, we instead take `multiplicityOverride :: Maybe Int` here, and set `isMultiplicity = fromMaybe 2 multiplicityOverride`?

Reply:
Ah, this seems to conflict a bit with this approach to testing capacity limits:
https://github.com/IntersectMBO/ouroboros-consensus/blob/f2f942b4279582a4a5b296c04698d0a2e9c12c82/ouroboros-consensus/test/consensus-test/Test/Consensus/Mempool.hs#L28-L31

Reply:
Yeah. We're fully in sync without even talking yet :)

I'm thinking: one option to increase the block multiplicity (ie an argument to `max`) and one option to decrease the block capacity (ie an argument to `min`).